where-to-travel/src/main/webapp/login.js

```diff
@@ -1,21 +1,30 @@
 // Get elements for authentication
 const emailLogin = document.getElementById('emailLogin');
 const passwordLogin = document.getElementById('passwordLogin');
-const btnLogin = document.getElementById('login');
 
-// Add login event
-btnLogin.addEventListener('click', e => {
-  const email = emailLogin.value;
-  const pass = passwordLogin.value;
-  const auth = firebase.auth();
-  
-  // Login 
-  const promise = auth.signInWithEmailAndPassword(email, pass);
-  promise.then(e => {
-    alert("You have sucessfully logged in!");
-  });
-  promise.catch(e => {
-    console.log(e.message);
-    alert(e.message);
+// Preferred method of material form behavior interaction is jQuery
+// When the document is loaded, add validity check to form on submit.
+$(document).ready(function() {
+  $(document).on('submit', '#loginForm', function() {
+    const email = emailLogin.value;
+    const pass = passwordLogin.value;
```

Comment:
nit: prefer not shortening names, ->password?

Reply:
fixed!